ports/py-lxml/portfile.cmake

```diff
@@ -5,6 +5,8 @@ vcpkg_from_pythonhosted(
     SHA512          7f8a3717645893bc6f790cc9adfb8fdab91c352dc4dc23c0ccb4af9a0d138acf9ef5054e9786af497955f10079e9242dbd63ea9ac39c33bfd71ca2fe4ef4a7c0
 )
 
+set(ENV{INCLUDE} "${CURRENT_INSTALLED_DIR}/include;$ENV{INCLUDE}")
```

Comment:
Can we set this to windows only (`if(VCPKG_TARGET_IS_WINDOWS)`)?
It might also be necessary for other platforms, but in this case will require a `:` path separator.

Reply:
My inspiration was https://github.com/open-vcpkg/python-registry/blob/12042231cf08521d3b13904dd3ec5e316aa8008e/ports/py-matplotlib/portfile.cmake#L9-L10

Reply:
oh, that needs fixing too in this case

Reply:
https://github.com/open-vcpkg/python-registry/pull/67